Lib/email/headerregistry.py

```diff
@@ -31,6 +31,11 @@ def __init__(self, display_name='', username='', domain='', addr_spec=None):
         without any Content Transfer Encoding.
 
         """
+
+        inputs = ''.join(filter(None, (display_name, username, domain, addr_spec)))
+        if '\r' in inputs or '\n' in inputs:
+            raise ValueError("invalid inputs; address parts cannot contain CR / LF")
```

Comment:
Hmm.  Reading this I think I'd say "arguments" rather than inputs, that aligns better with our typical vocabulary.  And how about "CR or LF"?